libr/core/cconfig.c

```diff
@@ -51,28 +51,28 @@ static void print_node_options(RConfigNode *node) {
 }
 
 static int compareName(const RAnalFunction *a, const RAnalFunction *b) {
-	return a && b && a->name && b->name && strcmp (a->name, b->name);
+	return (a && b && a->name && b->name ?  strcmp (a->name, b->name) : 0);
 }
 
 static int compareNameLen(const RAnalFunction *a, const RAnalFunction *b) {
-	return a && b && a->name && b->name && strlen (a->name) > strlen (b->name);
+	return (a && b && a->name && b->name  ? strlen (a->name) - strlen (b->name) : 0);
```

Comment:
this and all the following functions should use the usual body: `(a > b) - (a < b)`

Reply:
Fixed and rebased on master.